### ISSUE TITLE
What's the data of this?

### ISSUE THREAD

**Issue by lucasjinreal** [qianqiao/seq2seq#1]
Is this weibo data or something else? Could you give me a link of that?

**Comment by qianqiao** [comment]
sorry, the data is private for us. you could use any chat data to train this model. : )





--

清华大学计算机系       钱桥

电子邮箱：qianqiaodecember29@126.com

At 2017-03-07 14:08:49, "JinTian" <notifications@github.com> wrote:


Is this weibo data or something else? Could you give me a link of that?

**Comment by lucasjinreal** [comment]
Sorry, I think this is kind of private repo, I'll not using this for commercial use, I just want figure out the pipeline of seq2seq model, Would you like show me some sample in xxx.post and xxx.reply? Or just lines of questions and answers?

**Comment by qianqiao** [comment]
Of course!  the example file is in attachment. hope it is helpful for you : )




--

清华大学计算机系       钱桥

电子邮箱：qianqiaodecember29@126.com

At 2017-03-07 15:01:07, "JinTian" <notifications@github.com> wrote:


Sorry, I think this is kind of private repo, I'll not using this for commercial use, I just want figure out the pipeline of seq2seq model, Would you like show me some sample in xxx.post and xxx.reply? Or just lines of questions and answers?

**Comment by lucasjinreal** [comment]
Thanks so much!

**Comment by zhanghainan** [comment]
could you please give us a link of the examples? I can't find the example files. 
[VAE DIALOG.pdf](https://github.com/qianqiao/seq2seq/files/860289/VAE.DIALOG.pdf)
Thanks

**Comment by lkluo** [comment]
still can not find example file.
